### DISCUSSION TITLE
Feedback for Visual Studio Code Setup

### DISCUSSION THREAD

**Discussion by jcollard** [CaptainCoderOrg/IntroToCSharpSite#116] [Page Feedback]
Feedback discussion for Visual Studio Code Setup: https://csharp-develop.captaincoder.org/getting-started/vs-code

**Comment by cloudthepain** [comment]
Note: If you are not logged in or if you are using a free tier account, compilation may take 15 - 20 seconds.
>>> If you are using a free replit it will take 15-20 seconds to...

**Reply by jcollard** [reply]
This was actually on the "Book Features" section. Thanks! :)
